src/main/java/uk/org/tombolo/importer/ons/ONSWellbeingImporter.java

```diff
@@ -0,0 +1,160 @@
+ *  "attribute": {
+ *   "provider": "uk.gov.ons",
+ *   "label": "life_satisfaction"
+ *   }
+ * }
+ */
+public class ONSWellbeingImporter extends AbstractONSImporter {
+
+    private static final Logger log = LoggerFactory.getLogger(ONSWellbeingImporter.class);
+
+    private enum DatasourceId {
+        ONSWellbeing(new DatasourceSpec(
+                ONSWellbeingImporter.class,
+                "ONSWellbeing",
+                "Headline estimates of personal well-being at Local Authority level",
+                "Self-perception of Life satisfaction, Self-perception of Anxiety, Self-perception of Happiness, Self-perception of Worthiness",
+                "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/wellbeing/datasets/headlineestimatesofpersonalwellbeing/april2016tomarch2017localauthorityupdate/headlinewellbeinglocalauthorityupdate2016to2017.xls")
+        );
+
+        private DatasourceSpec datasourceSpec;
+        DatasourceId(DatasourceSpec datasourceSpec) {
+            this.datasourceSpec = datasourceSpec;
+        }
+    }
+    public ONSWellbeingImporter(){
+        datasourceIds = stringsFromEnumeration(DatasourceId.class);
+    }
+
+    @Override
+    public DatasourceSpec getDatasourceSpec(String datasourceId) throws Exception {
+        return DatasourceId.valueOf(datasourceId).datasourceSpec;
+    }
+
+    @Override
+    protected List<String> getOaDatasourceIds() {
+        return Arrays.asList(
+                OaImporter.OaType.localAuthority.datasourceSpec.getId(),
+                OaImporter.OaType.englandBoundaries.datasourceSpec.getId());
+    }
+
+    @Override
+    protected void importDatasource(Datasource datasource, List<String> geographyScope, List<String> temporalScope, List<String> datasourceLocation) throws Exception {
+
+        String fileLocation = DatasourceId.ONSWellbeing.datasourceSpec.getUrl();
+        InputStream isr = downloadUtils.fetchInputStream(new URL(fileLocation), getProvider().getLabel(), ".xls");
+
+        // This dataset contains both subject types
+        SubjectType localauthority = SubjectTypeUtils.getSubjectTypeByProviderAndLabel(AbstractONSImporter.PROVIDER.getLabel(), getOaDatasourceIds().get(0));
+        SubjectType englandboundaries = SubjectTypeUtils.getSubjectTypeByProviderAndLabel(AbstractONSImporter.PROVIDER.getLabel(), getOaDatasourceIds().get(1));
+
+        List<TimedValue> timedValues = new ArrayList<TimedValue>();
+
+        HSSFWorkbook workbook = new HSSFWorkbook(isr);
+        int sheetIndex = 0;
+
+        // Looping through the excell sheets
+        for (int sheet = 1; sheet <= 8; sheet = sheet+2){
+            Sheet datatypeSheet = workbook.getSheetAt(sheet);
+
+            // Creating the row iterator object
+            Iterator<Row> rowIterator = datatypeSheet.rowIterator();
+            // Skipping unrelevant rows
+            int ignore = 0;
+            while (ignore++ < 7) {
+                rowIterator.next();
+            }
+
```

Comment:
Add a log.info() here to tell the user which format is the date and how we store it.